app/main/views/g_cloud.py

```diff
@@ -340,54 +332,67 @@ def save_search(framework_framework):
         **build_search_query(request.args, filters.values(), content_manifest, lots_by_slug)
     )
 
+    projects = get_direct_award_projects(current_user.id)['open_projects']
+    projects.sort(key=lambda x: x['name'])
```

Comment:
`key=itemgetter('name')` is perhaps slightly neater? never quite convinced either way :)